Kernel/CMakeLists.txt

```diff
@@ -432,32 +423,12 @@ if (ENABLE_KERNEL_ADDRESS_SANITIZER)
 endif()
 
 add_compile_definitions(KERNEL)
-
-# HACK: This is a workaround for CLion to grok the kernel sources.
-#       It's needed because CLion doesn't understand the way we switch compilers mid-build.
-add_compile_definitions(__serenity__)
```

Comment:
Was this lost? I can't find it..

Reply:
In theory it's not required anymore. The Build/ARCH/CMakeToolchain.txt file should make CLion pick up the correct compiler from the start. I don't have a CLion install to test that theory though :)